### PR TITLE
Update for new design of MLO daily

### PR DESCRIPTION
**Pull request by Alixers505**
https://www.pivotaltracker.com/story/show/173481132

Screen Shot 2020-06-29 at 11 57 55 AM
Screen Shot 2020-06-29 at 11 57 50 AM
Screen Shot 2020-06-29 at 11 57 43 AM
Screen Shot 2020-06-29 at 3 05 34 PM

### CODE REVIEW

**Comment by Alixers505** [comment]
@B77Mills or @CaitlynKnudsvig - If either of you have time today, could you review this? The brand requested a release date of June 30th. Gratzi!

File: packages/common/components/style-b/blocks/title-only.marko [hunk 1]
```diff
@@ -12,15 +12,12 @@ $ const {
   limit,
   skip,
 } = input;
-$ const titleTableStyle = defaultValue(input.titleTableStyle, "font-family: Garamond, serif; font-size: 16px; line-height: 24px; margin: 0 auto; table-layout: fixed; mso-table-lspace: 0pt; mso-table-rspace: 0pt; color: #000;");
-$ const titleStyle = defaultValue(input.titleStyle, "font-family:Arial, 'Helvetica Neue', Helvetica, sans-serif; font-size:16px; margin-top: 9px; margin-left: 20px; margin-bottom: 9px; font-weight: bold;");
-$ const mainTableStyle = defaultValue(input.mainTableStyle, "border-collapse:collapse; font-family: Garamond, serif; font-size: 16px; line-height: 24px; margin: 0; mso-table-lspace: 0pt; mso-table-rspace: 0pt;");
+$ const titleTableStyle = defaultValue(input.titleTableStyle, "font: 300 16px/24px Arial, 'Helvetica Neue', Helvetica, sans-serif; margin: 0 auto; table-layout: fixed; mso-table-lspace: 0pt; mso-table-rspace: 0pt; color: #000;");
```
Comment (CaitlynKnudsvig) [inline packages/common/components/style-b/blocks/title-only.marko:17]: Shouldn't the font-weight be 400?  I don't think Arial renders Light/300

File: tenants/mlo-online/templates/daily.marko [hunk 2]
```diff
@@ -1,35 +1,45 @@
...
-$ const titleTableStyle = titleStyle + "table-layout: fixed; mso-table-lspace: 0pt; mso-table-rspace: 0pt; width: 100%; color: #000; background-color: #fff;";
-$ const featuredMainTableStyle = "background-color: #ebebeb;";
+$ const titleStyle = "font: bold 20px/30px 'Helvetica Neue', Helvetica, sans-serif; margin: 0; padding: 5px 20px;";
+$ const featuredTitleStyle = "font: bold 20px/30px 'Helvetica Neue', Helvetica, sans-serif; margin: 0; background-color: #307ED3; padding: 5px 20px; color: #fff; text-transform: uppercase;";
+$ const titleTableStyle = titleStyle + "table-layout: fixed; mso-table-lspace: 0pt; mso-table-rspace: 0pt; width: 100%; color: #555; background-color: #fff;";
+$ const featuredTitleTableStyle = titleStyle + "table-layout: fixed; mso-table-lspace: 0pt; mso-table-rspace: 0pt; width: 100%; color: #555; background-color: #307ED3; padding: 0;";
+$ const featuredMainTableStyle = "border-collapse:collapse; font: 300 16px/24px Arial, 'Helvetica Neue', Helvetica, sans-serif; margin: 0; mso-table-lspace: 0pt; mso-table-rspace: 0pt; background-color: #eee;"
```
Comment (CaitlynKnudsvig) [inline tenants/mlo-online/templates/daily.marko:15]: Ditto font-weight question here

File: packages/common/components/style-b/blocks/title-only.marko (hunk 1 above)
Comment (Alixers505) [reply packages/common/components/style-b/blocks/title-only.marko:17]: yes, thank you

**Pull request merged**